packages/coinjoin/src/client/middleware.ts

```diff
@@ -104,8 +104,17 @@ export const getAnonymityScores = async (
     transactions: AnalyzeTransactionDetails[],
     options: RequestOptions,
 ) => {
-    const data = await request<AnalyzeResult>('get-anonymity-scores', { transactions }, options);
-    return data.results;
+    try {
+        const data = await request<AnalyzeResult>(
+            'get-anonymity-scores',
+            { transactions },
+            options,
+        );
+
+        return data.results;
+    } catch (error) {
+        return null;
```

Comment:
log the error here please

Reply:
Not console.log, but coinjoin logger should be used here. Probably better to wait for https://github.com/trezor/trezor-suite/pull/7603/ to get merged and then user the new logger API right away.

Reply:
bff1f488f

Reply:
I don't think we need/want to report accountKey to sentry, not even redacted.

Reply:
why not to catch this error in `analyzeTransactions` module?
this will be the only case in `middleware` module with try/catch wrapper and you need to pass `accountKey` and `logger` to use it here

Reply:
3b6c309a0